scripts/lib/mkcloud-libvirt.sh

```diff
@@ -103,6 +103,11 @@ function libvirt_prepare()
 
 function libvirt_setupadmin()
 {
-    ${mkcloud_lib_dir}/libvirt/admin-config $cloud $admin_node_memory $adminvcpus $emulator $admin_node_disk "$localreposdir_src" "$localreposdir_target" > /tmp/$cloud-admin.xml
+    nvramtemplate=""
```

Comment:
Yes, because the template might be distribution specific and by not hardcoding we just use the autodetected default which should "work" always.